src/utils/isAfterDay.js

```diff
@@ -0,0 +1,12 @@
+import moment from 'moment';
+
+export default function isAfterDay(a, b) {
+  if (!moment.isMoment(a) || !moment.isMoment(b)) return false;
+
+  const isSameYear = a.year() === b.year();
```

Comment:
You might be able to squeeze out a bit more perf by storing the results of x.year/month/date, depending on how expensive those calls are and how often this function is called.